core/src/main/java/com/linecorp/armeria/server/ServiceBuilder.java

```diff
@@ -0,0 +1,123 @@
+package com.linecorp.armeria.server;
+
+import java.nio.charset.Charset;
+import java.time.Duration;
+import java.util.function.Function;
+
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.logging.ContentPreviewer;
+import com.linecorp.armeria.common.logging.ContentPreviewerFactory;
+import com.linecorp.armeria.server.logging.AccessLogWriter;
+
+interface ServiceBuilder {
```

Comment:
How about renaming `ServiceBuilder` to `ServiceConfigSetters` and `DefaultServiceBuilder` to `DefaultServiceConfigSetters`? WDYT, @minwoox @ikhoon ?

Reply:
That's an excellent idea! I love that name. :-) This class only used for configuring service.